great-idea/services.html

```diff
@@ -31,13 +33,14 @@
 <div class="container">
 <!-- Hero Section -->
     <hero class="short-hero">
-		<img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
-
-		<h2>Services</h2>
-		<p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus
-		rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus
-		et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod
-		facilisis iaculis.</p>
+			<img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+			<div class="hero-content">	
+				<h2>Services</h2>
+				<p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus
+				rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus
+				et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod
+				facilisis iaculis.</p>
+			</div>
     <!-- End Hero Section -->
 	</hero>
 	<div class="divider-line"><br></div>
```

Comment:
You could also use a [horizontal rule](https://devdocs.io/html/element/hr) here instead of a div, which provides a bit more semantic meaning.